pkg/operator2/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -58,8 +58,10 @@ func NewConfigObserver(
 					configInformer.Config().V1().Infrastructures().Informer().HasSynced,
 				),
 			},
-			apiserver.ObserveAdditionalCORSAllowedOrigins,
-			apiserver.ObserveTLSSecurityProfile,
+			apiserver.NewCORSObserver(configobservation.OAuthServerConfigPrefix),
+			apiserver.NewCORSObserver(configobservation.OAuthAPIServerConfigPrefix),
+			apiserver.NewTLSSecurityProfileObserver(configobservation.OAuthServerConfigPrefix),
```

Comment:
the requirements have already changed and we would like to get rid of one unnecessary layer (a separate config struct) and pass those directly as flags to API servers ( we will probably refactor openshift-apiserver too).

For example, they should end up in https://github.com/openshift/cluster-authentication-operator/pull/243/commits/f35f38ef46f77ba117bae4e011cd2349fc31b1c2#diff-5f7fcb2a076015ed36e46e8c49d557c8R23